configs/experiments/ood_blur_coco_gaussian.yaml

```diff
@@ -0,0 +1,18 @@
+exp:
+  name: ood_gaussian_coco_people_gaussian
+  log_dir: logs
+model:
+  weights: weights/coco_people_gaussian/version_0/state_dict_best_mae.ckpt
+  test_cfg: configs/test/coco_gaussian_cfg.yaml
+data:
+  test_examples: 10
+  n_samples: 5
+  perturb: {'mean': 5.0, 'std': 5.0}
+  module: ood_blur_coco_people # {ood_blur_coco_people, ood_mixup_coco_people, coco_people}
+hyperparams:
+  lmbda: 0.01
+  x_kernel_gamma: 5.0
```

Comment:
Are you using the RBF kernel for these experiments? Or polynomial? I thought the MMD paper we found used 1/d as the gamma value for polynomial

Reply:
Yeah good call out - these hyperparms weren't even actually feeding into any code. Fixed.

Reply:
I also installed and ran the pre-commit hook and linter